### ISSUE TITLE
[Typescript] Property 'malarkey' is private and only accessible within class 'MalarkeyController'.

### ISSUE THREAD

**Issue by dereklin** [Swiip/generator-gulp-angular#993]
I am running version 1.0.2.  I was testing a project with typescript.  When I do gulp serve, I get this error:

```
Property 'malarkey' is private and only accessible within class 'MalarkeyController'.
```

It might be related to https://github.com/Swiip/generator-gulp-angular/pull/936

Is changing the variable from private to public the right fix?

**Comment by kuncevich** [comment]
Got same thing in file .../src/app/components/malarkey/malarkey.directive.ts line 25 char 15

**Comment by jmfirth** [comment]
Ran into this as well.  Fixed in 8132cf7c39dd92690b8b3626c06d3dc9a06088cc
